src/Framework/Menus/CustomOrderBoard.cs

```diff
@@ -148,6 +148,7 @@ public override void snapToDefaultClickableComponent()
         public override void gameWindowSizeChanged(Rectangle oldBounds, Rectangle newBounds)
         {
             base.gameWindowSizeChanged(oldBounds, newBounds);
+            Game1.activeClickableMenu = new CustomOrderBoard(this.boardType, this._billboardTexture);
```

Comment:
Why?

Reply:
This is so the game reopens the menu when the game windows is resized, circumventing the issue that the buttons get displaced. 
![image](https://user-images.githubusercontent.com/25935015/117784174-8d69b480-b243-11eb-9137-af8c5d95982e.png)

The base game does it that way. I could also 
1) just update X/Y of the buttons, but then we either have hardcoded pixel values in 2 different places 
2) remove the change entirely and just let it be, window resizing will probably occur very rarely while the quest window is open.